### PR TITLE
#29

### PR DESCRIPTION
**Pull request by fionapigott**
- change behavior of get_entities for Retweets
- fixed a minor bug where get_generator didn't work for py2 (I noticed this while testing)
- version to 1.11.1 (minor bigfix)

- Tested for python 2 and 2
- doctests run

- built the new gh pages and pushed, you can check them out online

### CODE REVIEW

**Comment by fionapigott** [comment]
I tagged both of you, but I think only one review is fine.

File: tweet_parser/getter_methods/tweet_entities.py [hunk 1]
```diff
@@ -1,9 +1,17 @@
 from tweet_parser.tweet_checking import is_original_format
-
+from tweet_parser.getter_methods.tweet_embeds import get_retweeted_tweet
+from tweet_parser.getter_methods.tweet_text import get_tweet_type
 
 def get_entities(tweet):
     """
-    Helper function to simply grabbing the entities.
+    Helper function to simply grabbing the entities. \n
+    Caveat: In the case of Retweets, a Retweet is stored as
```
Comment (jrmontag) [inline tweet_parser/getter_methods/tweet_entities.py]: I think it's a good idea to be sure this is communicated to the user. Do you think it's better to do this here in the function docstring, or in the html documentation, or some combination of the two?

Comment (binaryaaron) [reply tweet_parser/getter_methods/tweet_entities.py]: The html docs are generated from the docstrings and can be seen [here](https://tw-ddis.github.io/tweet_parser/tweet_parser.getter_methods.html#module-tweet_parser.getter_methods.tweet_entities] - is there a visibility problem here that we should address? I'm open to creating some type of auxiliary HTML doc pages that define some assumptions like this if you think it's needed.

Comment (fionapigott) [reply tweet_parser/getter_methods/tweet_entities.py]: I thought that both of these sort of fit into the "minor bugfix" category--neither seemed like "new" functionality. But I'm open to changing that.

Comment (jrmontag) [reply tweet_parser/getter_methods/tweet_entities.py]: I wasn't clear in my question. The docstring-generating docs are great. I think it makes sense to add this (or some similar form of) "caveat" as @fionapigott has done here. 

My question was meant to be more hypothetical, probably for down the road, and not meant to hinder this PR: at what point would we consider creating a separation between docstrings and an auxiliary (presumably longer) set of docs. A motivating example from sklearn is the [2-3 sentences in this docstring](http://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.CountVectorizer.html#sklearn.feature_extraction.text.CountVectorizer) versus the longer exposition in the User's Guide.

Comment (binaryaaron) [reply tweet_parser/getter_methods/tweet_entities.py]: Per our around-the-table discussion today, let's keep the bugfix designation here.

@jrmontag - I will open another issue for improving the docs around some of our exceptions and edge cases as it's certainly worth making it more visible. Thanks for the suggestions.

**Pull request merged**